sdk/textanalytics/ai-text-analytics/samples-dev/entityLinking.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample uses the linked entity recognition endpoint to detect
+ * well-known entities in a document and connect (link) them to entries in an
+ * external knowledge base (such as Wikipedia) that contain information about
+ * the entity.
+ *
+ * @summary detects entities that have links to more information on the web
+ * @azsdk-weight 100
+ */
+
+import { TextAnalysisClient, AzureKeyCredential } from "@azure/ai-text-analytics";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+// You will need to set these environment variables or edit the following values
+const endpoint = process.env["ENDPOINT"] || "<cognitive services endpoint>";
```

Comment:
should this say cognitive language service endpoint instead?

Reply:
Good point, done.